### ISSUE TITLE
no-raw-colors flags non-color theme tokens (shadow-*, text-* size tokens)

### ISSUE THREAD

**Issue by justinchen** [shadcn-ui/lint#17]
`no-raw-colors` reports theme tokens that are not colours.

In a Tailwind v4 project whose `@theme inline` declares shadow and font-size tokens, using them is flagged:

```
"shadow-floating" is not a theme color. Use a token declared in src/app/styles/globals.css
```

### Repro

```css
@theme inline {
  --shadow-surface: 0 1px 2px rgb(0 0 0 / 0.06);
  --shadow-raised: 0 2px 6px rgb(0 0 0 / 0.08);
  --shadow-floating: 0 8px 24px rgb(0 0 0 / 0.12);
  --text-icon-lg: 1.5rem;
}
```

```tsx
<div className="shadow-floating text-icon-lg" />
```

Both are reported by `shadcn/no-raw-colors`.

### Expected

A `shadow-<name>` / `text-<name>` utility that resolves to a declared non-colour token (shadow, font-size) is not a raw colour — it is exactly the token usage the rule exists to encourage.

### Impact

On our codebase (~1,700 files under one App Router tree) this is **91 of 183** `no-raw-colors` findings: 90 shadow tokens plus `text-icon-lg`. We cannot enable the rule until these stop being reported.

Versions: `@shadcn/lint` 0.1.0, oxlint 1.83, Tailwind 4.3.
